lib/services/nativescript-project-plugins-service.ts

```diff
@@ -110,18 +110,23 @@ export class NativeScriptProjectPluginsService implements IPluginsService {
 
 	public addPlugin(pluginIdentifier: string): IFuture<void> {
 		return (() => {
+			let pluginBasicInfo: IBasicPluginInformation;
 			if(this.isPluginInstalled(pluginIdentifier)) {
-				this.$logger.printMarkdown(util.format("Plugin `%s` is already installed.", pluginIdentifier));
+				this.$logger.printMarkdown(util.format("Plugin `%s` is already installed.", pluginIdentifier.toLowerCase()));
 				return;
 			}
 
-			if(this.checkIsValidLocalPlugin(pluginIdentifier).wait()) {
-				this.installLocalPlugin(pluginIdentifier, {addPluginToPackageJson: true}).wait();
+			if (path.extname(pluginIdentifier) === ".tgz") {
+				this.fetch(pluginIdentifier).wait();
```

Comment:
hm, maybe I'm not correct, but `fetch` method installs the package to temp dir and copies it from there to plugins lib. After that it adds the plugin as local one from the plugins dir. So in the package.json you'll have:
```JSON
"dependencies": {
    "my-fetched-plugin": "file:plugins/myplugins"
}
```
Sooo this is not the expected behavior when I have `.tgz`. In fact we have to:
1) Try installing it to temp dir and check its package.json (get the id from package.json).
2) Check if the path to .tgz is in current project dir. If it is not, copy **the .tgz file** to the project dir
3) Add it to package.json using the file in project dir.

So we should receive something like:
```JSON
"dependencies": {
    "my-fetched-plugin": "file:plugins/myfile.tgz"
}
```